src/pages-and-resources/PagesAndResources.jsx

```diff
@@ -33,7 +31,6 @@ function PagesAndResources({ courseId, intl }) {
       <main className="container container-mw-md">
         <h3 className="mt-5 mb-5">{intl.formatMessage(messages.heading)}</h3>
         <PageGrid pages={pages} />
-        <ResourceList />
```

Comment:
@awais-ansari Since the ResourceList is not being used anymore we will need to remove the actual component and everything related as well